install.php

```diff
@@ -29,4 +29,4 @@
 $admin->email = "admin@concordia.ca";
 $admin->is_admin = 1;
 $admin->password_digest = password_hash('root', PASSWORD_DEFAULT);
-$admin->save();
\ No newline at end of file
+$admin->save();
```

Comment:
you'll need to add a default value for is_instructor here too

Reply:
do you think admin should also be an instructor by default or not?

Reply:
set it to false for now, can be changed later if we decide otherwise